pants-plugins/src/python/internal_backend/sitegen/rules.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
Could we delete this file instead?

Reply:
Ooops not sure how that snuck in there, it's unrelated. Deleting.